README.md

```diff
@@ -15,11 +15,10 @@
 
 ## Getting started
 
-The app requires Xcode 9.4 to build and run, which can be obtained from the Mac App Store. It also uses [CocoaPods](http://cocoapods.org/) to integrate third-party frameworks.
-
-1. Install CocoaPods on your machine.
+1. Install [CocoaPods](http://cocoapods.org/).
 2. Run `pod install` in the root folder of the repository.
-3. Open `PrideFestival.xcworkspace` (instead of `PrideFestival.xcodeproj`).
+3. Obtain the files `fabric.apikey` and `fabric.buildsecret` and add them to the root of the project. They can also be created with your own keys, or you can remove Fabric integration.
```

Comment:
How hard would it be to either make the staging build not use Fabric, or to have a separate "dev" build that didn't need the keys?

Reply:
And I guess, what would we lose if drive-by contributors didn't have the ability to build with Fabric integration?

Reply:
I haven't really thought about removing Fabric from the staging build, but that makes a lot of sense. I have a feeling it won't be hard. We aren't really using Fabric for staging anymore. Contributors don't need Fabric integration.